projects/aca-playwright-shared/src/api/favorites-api.ts

```diff
@@ -115,4 +115,24 @@ export class FavoritesPageApi {
       return await Utils.retryCall(favoriteFiles);
     } catch {}
   }
+
+  async removeFavoriteById(username: string, nodeId: string): Promise<void> {
```

Comment:
This functions are almost the same, let's just have one to remove multiple favorites and if you need to remove one you can do it like: `[favoriteId]`